src/lib/operators/jit_operator/operators/jit_abstract_sink.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
```

Comment:
What's up with the folder structure? What does the last "operators" mean? How about `src/lib/operators/jit` or `src/lib/jit/operators`?

Reply:
PR4 will introduce `src/lib/operators/jit_operator.cpp/hpp`. The idea was to bundle all of its internals in `src/lib/operators/jit_operator/...` similar to `src/lib/operators/table_scan`. But we can go for a different folder structure if you prefer.
I think there is a general naming problem with the term jit operator. It can refer to the wrapping `JitOperator` (that inherits from `AbstractReadOnlyOperator` and is an operator from the PQPs point of view) and the operators inside the `JitOperator` such as `JitCompute`, `JitFilter` etc. But I don't have any good alternative name for either of them.

Reply:
Good thing that you bring this up. We need to find a solution for this, because otherwise people will get confused. I like having a normal `Aggregate` and a `JitAggregate`. The question is about `JitAbstractOperator`, where people would rightfully expect it to inherit from `AbstractOperator`.

I was thinking about `JitStep` for a while, but that might get confused with the steps that you take to arrive at the `Jit(ted)Operator`.

Something like `JitIntegrateable`, `AbstractJittable`?

Reply:
I like `AbstractJittable`. We also have `JitAbstractSink` (which provides a common interface for the last operators in a jit chain). I would suggest `AbstractJittableSink` for that then.
And finally there is the `JitOperator` itself that inherits from `AbstractOperator`. Can that stay as it is/ I was thinking about `JitOperatorWrapper`/`Container`/`Adapter`.